lib/config/executor/container_vm.go

```diff
@@ -134,6 +134,9 @@ type ExecutorConfig struct {
 	// Layer id that is backing this container VM
 	LayerID string `vic:"0.1" scope:"read-only" key:"layerid"`
 
+	// Blob metadata for the caller
+	Annotations map[string]string `vic:"0.1" scope:"hidden" key:"annotation"`
```

Comment:
We're not hardcoding laye id version.  This is a golang hint to allow encoders/decoders to serialize/deserialize golang structs.